components/atoms/Button/button.tsx

```diff
@@ -23,7 +23,7 @@ const Button = React.forwardRef<HTMLButtonElement, ButtonsProps>(
           props.variant === "outline" && style.outline,
           props.variant === "link" && style.link,
           disabled && "bg-light-orange-7 hover:bg-light-orange-7 border-none pointer-events-none",
-          "inline-flex text-sm font-semibold tracking-tight border py-2 px-4 rounded-md focus-visible:border-orange-500 focus:outline-none focus-visible:ring focus-visible:ring-orange-200"
+          "inline-flex text-sm font-semibold tracking-tight border py-2 px-4 rounded-md focus-visible:border-orange-500 focus:outline-none focus-visible:ring focus-visible:ring-orange-200 whitespace-nowrap"
```

Comment:
happy with changing all buttons to encompass this change, tho it should be checked in the storybook as well